scapy/arch/bpf/core.py

```diff
@@ -70,9 +70,9 @@ def get_if_raw_hwaddr(ifname):
         raise Scapy_Exception("Failed to execute ifconfig: (%s)" % msg)
 
     # Get MAC addresses
-    addresses = [l for l in fd.readlines() if l.find("ether") >= 0 or
-                 l.find("lladdr") >= 0 or
-                 l.find("address") >= 0]
+    filter_line = lambda l: l.find("ether") >= 0 or l.find("lladdr") >= 0 or \
+        l.find("address") >= 0
+    addresses = [l for l in fd.readlines() if filter_line(l)]
```

Comment:
On this one, I'd rather keep the old code, which looks more readable to me.

Reply:
Done.
I had to use `noqa`.